jenkins_pipelines/scripts/json_generator/maintenance_json_generator.py

```diff
@@ -215,12 +215,8 @@ def parse_cli_args() -> argparse.Namespace:
     return args
 
 def read_mi_ids_from_file(file_path: str) -> list[str]:
-    try:
-        with open(file_path, 'r') as file:
-            return file.read().strip().split()
-    except Exception as e:
-        logging.error(f"Error reading file: {e}")
-        return []
+    with open(file_path, 'r') as file:
+        return file.read().strip().split()
```

Comment:
Why exception is no longer needed? What would happen if we get let say permission denied?

Reply:
The previous version captured all possible exceptions, logged an error and returned an empty list.
This meant that any error occurring with the file would not stop the script execution.

IMO is better to just let any exception regarding the file (permission issues, file not found...) propagate up the stack and have the script crash in such cases, so it is evident there's an issue and we do not risk using the results as if everything worked fine when it actually did not.

Reply:
Theoretically right, every exception is already handled in built-in. I just wanted to be reassured that this step would not pass in any border case like corrupted records, file but exception not thrown. But it should have been already done in the previous solution.

Resolving